### PR TITLE
example: translation use dot notation

### PR DESCRIPTION
**Pull request by zikowang**

### CODE REVIEW

File: src/i18n/locales/de.json [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
+  },
+  "nav": {
+    "newVocab": "Neue Vokabeln",
+    "learn": "Lernen",
+    "progress": "Fortschritt",
+    "allVocabulary": "Alle Vokabeln",
+    "groupLearning": "Gruppe lernen",
+    "settings": "Einstellungen"
+  }
 }
```
Comment (luwol03) [inline src/i18n/locales/de.json:14]: could you please delete the german translation from this pr? You can add it later on localazy if this pr is merged into the main branch.

**Review by luwol03 — CHANGES_REQUESTED** [review]

**Comment by noctera** [comment]
As a result of further work, merge conflicts occurred in the files. I will fix them as fast as I can.

File: src/i18n/locales/de.json (hunk 1 above)
Comment (zikowang) [reply src/i18n/locales/de.json]: I've reverted my changes to the German translation file.

**Review by noctera — COMMENTED** [review]
reviewed

**Review by luwol03 — APPROVED** [review]

**Pull request merged**